pkgs/by-name/pd/pdf2odt/package.nix

```diff
@@ -49,6 +49,10 @@ resholve.mkDerivation rec {
       imagemagick
       zip
     ];
+    execer = [
+      # zip can exec; confirmed 2 invocations in pdf2odt don't
+      "cannot:${zip}/bin/zip"
```

Comment:
The global override was a mistake (but I'm glad you're scrubbing hard enough to notice).

When going back over it, I noticed user-controlled exec in the `-TT cmd/--unzip-command cmd` options:
> Use command cmd instead of 'unzip -tqq' to test an archive when the -T option is used."

Reply:
In general, there's an open problem here about how to ~operationalize this kind of manual research by documenting what we looked at, what we found, and pinning ~assertions about the command(s) to the underlying source in a way that we can re-evaluate after updates that merit a fresh look. I imagined an advantage of keeping this stuff in the binlore repo was going to be having an obvious place to try to collect that knowledge, but I wasn't looking forward to figure out the right mix of automation/coupling to keep up with nixpkgs from there.

If the nixpkgs immune system doesn't reject this current override effort I'm thinking we might be able to figure out how to add passthru tests that assert knowledge about the underlying source for a CLI. I haven't put too much thought into that yet since it would be moot if this one's a nonstarter.